tools/wptserve/tests/functional/test_cookies.py

```diff
@@ -1,9 +1,8 @@
-import unittest
+from .base import TestUsingServer
 
 import pytest
 
 wptserve = pytest.importorskip("wptserve")
-from .base import TestUsingServer
 
 
 class TestResponseSetCookie(TestUsingServer):
```

Comment:
If we move to pytest fixtures, I think we can also remove the class wrappers here and just use normal functions.